setup.cfg

```diff
@@ -16,16 +17,22 @@ classifiers =
     Programming Language :: Python :: 3.5
     Topic :: Software Development :: Quality Assurance
 
-[files]
-packages =
-    edx_lint
-    edx_lint.cmd
-    edx_lint.pylint
+[options]
+packages = find:
+install_requires =
+    pylint==1.7.6
+    pylint-django==0.7.2
+    pylint-celery==0.3
+    six>=1.10.0,<2.0.0
+    click>=6.0
+    click-log==0.3.2
```

Comment:
How does this list tap into the pip-compile workflow? Or it doesn't?

Reply:
It does for `dev.in`/`dev.txt`. However, my expectation is that since edx-lint is intended to standardize pylint versions, that `setup.py` should pin particular pylint versions.

Reply:
It feels like we need a note someplace that these lines are outside the "make upgrade" flow.  How will people know to update these pins?

Reply:
I just added a message at the end of `make upgrade`